Fabric.Authorization.AccessControl/src/app/services/fabric-auth-user.service.ts

```diff
@@ -24,6 +25,15 @@ export class FabricAuthUserService extends FabricBaseService {
     if (!FabricAuthUserService.userRolesApiUrl) {
       FabricAuthUserService.userRolesApiUrl = `${FabricAuthUserService.baseUserApiUrl}/{identityProvider}/{subjectId}/roles`;
     }
+
+    if (!FabricAuthUserService.userGroupsApiUrl) {
+      FabricAuthUserService.userGroupsApiUrl = `${FabricAuthUserService.baseUserApiUrl}/{identityProvider}/{subjectId}/groups`;
+    }
+  }
+
+  public getUserGroups(identityProvider: string, subjectId: string) : Observable<string[]> {
+    return this.httpClient
+      .get<string[]>(this.replaceUserIdSegment(FabricAuthUserService.userGroupsApiUrl, identityProvider, subjectId));
```

Comment:
Same comment as for groups, do we need to urlencode the subjectId as these may have `\` in them.